interview-questions.md

```diff
@@ -17,52 +17,90 @@ INSTRUCTOR EXAMPLE: What is a conditional statement?
 
   Your answer:
 
+  Git is a version control program that occurs locally and remotely, meaning on your personal device and on the internet respectively. Github is a remotely occurring website that communicates with Git technology where it is possible to create folders known as repositories to store and share files and their version history as you make and update them.
+
   Researched answer:
 
+  Git's version control can happen locally without even having to access Github. Github is just a common cloud based host that enables sharing and and collaborating using Git. Multiple parties can collaborate on the same version of a project and save staged changes without changing the living version of a project until their changes are merged by a project administrator.
 
 
 2. Which JavaScript operators will return a Boolean value?
 
   Your answer:
 
-  Researched answer:
+  Boolean is the term used for true or false statements. There are three operators that will return a Boolean value.  
+
+  The loosely equals operator, denoted by two consecutive equals signs,== is more flexible with its definition of equivalence. For example if the operator was to be applied to the string of characters "42" and the numerical value of 42, the Boolean value would be true due to them both being 42, despite being different data types.
 
+  The strictly equals operator, represented by === is a lot pickier and will only return true if the data type and the value are the same. In the same example of "42" === 42, strictly equals would return false. Interestingly enough if a the variable assessmentNumber were to be assigned the value of 42, the variable itself would take on that new value and assessmentNumber === 42 would return a true value.
+
+  The "not operator" or the "Bang! operator," written !== (the exclamation mark replaces one of the equals in ===) will be true if the values being compared DO NOT match.
+
+  Researched answer:
 
+  It looks like I thought this questions was specifically about Equality Operators and I completely forgot about Relational Operators and Logical operators. The Relational Operators are >, <, <=, and >=. The Logical Operators are ||, &&, and !. All of these are capable of returning a Boolean value as long as the variable isn't undefined.
 
 3. What is an index? What is the difference between index and value?
 
-  Your answer:
+  ---Your answer:
 
-  Researched answer:
+  An index is the position number of information in an array or string counting from left to right and including 0. Including zero in the counting is called Zero Based Indexing. A value is just defined information but, when applied to an array, a value is the information contained at the specified index location.
+
+  Example: ["Eggs","Bread","Coffee","Milk","Cheese"]
```

Comment:
Appreciate the examples. This definitely shows your understanding. Good technique for interviews.